Answers.md

```diff
@@ -1,11 +1,16 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+React is a jvascript framework that renders a virtual DOM. DOM manipulation is problematic, and can cause bugs when things happen on the app. React takes objects and causes them to interact with eachother smoothly when things change on the app. 

```

Comment:
React is a JS library. It solves the problem of keeping track of DOM changes and making them more efficient